### ISSUE TITLE
Html file

### ISSUE THREAD

**Issue by chronoss09** [RetroGamer74/ESP8266_Homebrew_2.0_Switch_Pegaswitch#4]
can you post the source of the file please ?

**Comment by jose12o9** [comment]
yes please , if you don't plan to add the support for 1.0.0-3.0.0
at least publish the source to make the bin.

**Comment by jacobfernando** [comment]
Hay alguna manera de implementar evalfile usefulscripts/hbmenu.js al esp8266 para los firmware antes del 3.0.0?

**Comment by jose12o9** [comment]
chronoss : you  an grab the html files with some web grabber and 3 files more in loader.js

04/06/2018  05:32             1 868 exploit19ad.html
04/06/2018  05:32             1 860 exploita531.html
04/06/2018  05:32             1 671 index.html
04/06/2018  05:44           270 336 installer.nro
04/06/2018  05:32             7 316 loader.js
04/06/2018  05:33                 0 ps4icon.png
03/06/2018  07:55         1 916 528 rop.bin
03/06/2018  07:55           479 144 rop_relocs.bin
04/06/2018  05:32             3 884 settings.html

mayby it's a good start,
